src/pages/whats-new.js

```diff
@@ -0,0 +1,203 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { css } from '@emotion/core';
+import SEO from '../components/seo';
+import PageTitle from '../components/PageTitle';
+import { graphql } from 'gatsby';
+import { Layout, Link } from '@newrelic/gatsby-theme-newrelic';
+
+const MOBILE_BREAKPOINT = '960px';
+
+const WhatsNew = ({ data }) => {
+  const { allMarkdownRemark } = data;
+  const posts = allMarkdownRemark.edges.map(({ node }) => node);
+  const postsByDate = Array.from(
+    posts
+      .reduce((map, post) => {
+        const key = post.frontmatter.releaseDate;
+
+        return map.set(key, [...(map.get(key) || []), post]);
+      }, new Map())
```

Comment:
Out of curiosity, why did you reach for `Map` here?

Reply:
One of the differences between a `Map` and a regular object is that key order is preserved. In this case, its imperative that we maintain the sort order on the posts so that the dates are reliably sequential.

Reply:
Oh neat, I don't think I've ever used `Map` before. That's fantastic! 🎉